src/components/User-Landing-Page/User-Landing.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react'
+
+function UserLandingPage() {
+  const [user, setUser] = useState({id: 1, name: 'John', address: '123 Wherever St, Denver CO 80202', connections: false, about: 'My about me'})
+  const [connection, setConnection] = useState({id: 2, name:'Bill', about:'Howdy Im Bill'})
+
+  useEffect(() => {
+    console.log(user.connections)
+  }, [])
+
+  // PASS USER DATA THRU AS PROPS AFTER LOGIN, OR CALL THEM ON PAGE LOAD?
```

Comment:
Good question. I liked the discussions we had, but I'm still not sure. I feel like I'm leaning towards passing down as props after login?